### PR TITLE
Added 2 files

### PR DESCRIPTION
**Pull request by Pooja-Gangavathi**

### CODE REVIEW

File: emptyspaces.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//To count number of empty spaces in a string
+const string = " Stockholm IT Academy, Stockholm ";
+var count  = 0;
+for(i=0;i<string.length;i++){
+    if(string[i]==" "){
+        count++;
+    }
+}
+console.log("Number of spaces in a string is "+ count);
```
Comment (Ezhilarasi28) [inline emptyspaces.js:9]: It looks good.

File: string.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+//Accessing the last letter of the string without using index number
+console.log(string[string.length-1]);
+
+//To count how many letters are there in given string
+var count=0;
+for(i=0;i<string.length;i++){
+    count++;
+}
+console.log("Number of letters in a given string : " + count);
```
Comment (Ezhilarasi28) [inline string.js:20]: It looks good

Comment (Pooja-Gangavathi) [reply string.js:20]: Can you approve it? maybe I can merge it

Comment (Ezhilarasi28) [reply string.js:20]: ya sure

**Review by Ezhilarasi28 — COMMENTED** [review]
Good one.

**Pull request closed without merging**